src/components/form/components/FormDateTime.tsx

```diff
@@ -1,25 +1,16 @@
 import React from 'react';
-import { LocalizationProvider } from '@mui/x-date-pickers';
-import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
-import {
-  DateTimePicker,
-  DateTimePickerProps,
-} from '@mui/x-date-pickers/DateTimePicker';
+import * as Bootstrap from 'react-bootstrap';
```

Comment:
This is importing all of react-bootstrap (namespaced import). Might be worth importing just what we want instead, this will help reduce the size of the bundled package:

```suggestion
import { FormControlProps } from "react-bootstrap/FormControl";
import BootstrapForm from "react-bootstrap/Form";
```